### PR TITLE
fundi

### PR DESCRIPTION
**Pull request by kwarambatendai**

### CODE REVIEW

File: theme/lms/templates/static_templates/about.html [hunk 1]
```diff
@@ -10,7 +10,7 @@ <h1>
             <%block name="pageheader">${page_header or _("About")}</%block>
         </h1>
         <p>
-            <%block name="pagecontent">{{ PLATFORM_NAME }} is an online learning platform powered by <a href="https://open.edx.org">Open edX</a>. It runs with <a href="https://docs.tutor.overhang.io">Tutor</a>, with the customized <a href="https://github.com/overhangio/indigo">Indigo</a> theme.</%block>
+            <%block name="pagecontent">{{ PLATFORM_NAME }} is an online learning platform. Our oblective is to offer education content that is relevant to Afroca for Empowerment and upliftment.
```
Comment (regisb) [inline theme/lms/templates/static_templates/about.html:13]: Afroca -> Africa

**Comment by regisb** [comment]
I believe this PR was opened in error. Now closing.

Great customization btw :)

**Comment by kwarambatendai** [comment]
Thanks

**Comment by kwarambatendai** [comment]
Hi Regis.

Thanks for the great work you did on tutor.

I have an installation on fundiacademy.com however, I am stuck at
configuration of the ecommerce plugin which came with tutor.

Here is my question, do I need to build a shopping cart or there is one
already in the ecommerce plugin?

**Pull request closed without merging**